Lib/poplib.py

```diff
@@ -304,7 +304,7 @@ def rpop(self, user):
         return self._shortcmd('RPOP %s' % user)
 
 
-    timestamp = re.compile(br'\+OK.*(<[^>]+>)')
+    timestamp = re.compile(br'\+OK.[^<]*(<.*>)')
```

Comment:
Previous version matched OK<capturedvalue> but the change doesn't, as it demands a single character after the OK before looking for the <. Removing the "." after OK will restore the equivalence, if this is desired.